README.md

```diff
@@ -1786,9 +1807,36 @@ npm run lint
   }
   ```
 
+- [12.11](#12.11) <a name="12.11"></a> When declaring constant instance members in a constructor, prefer class property syntax.
+
+  ESLint rule: [`prefer-class-properties`](packages/eslint-plugin-shopify/docs/prefer-class-properties.md)
+
+  > Why? This often removes the need for an explicit constructor and clearly indicates "setup" members for a class. It also matches how type declarations for instance properties are done in Flow.
```

Comment:
You better watch out, keep providing good suggestions and I will make you write all my docs :P